tsdb/ooo_head_read_test.go

```diff
@@ -343,6 +327,7 @@ func TestOOOHeadIndexReader_Series(t *testing.T) {
 					if headChunk && len(intervals) > 0 {
 						// Put the last interval in the head chunk
 						s1.ooo.oooHeadChunk = &oooHeadChunk{
+							chunk:   NewOOOChunk(),
```

Comment:
This avoids crashing on a nil pointer, but is still unrealistic since an OOO chunk with zero samples should not be created in the real code.